pages/watch/[slug].tsx

```diff
@@ -1,15 +1,42 @@
 import { useRouter } from 'next/router'
-import React, { FC } from 'react'
+import React, { FC, useState, useEffect } from 'react'
+import firebase from 'firebase/app'
+import 'firebase/database'
+import { config } from '../../firebaseConfig'
+import { useObjectVal } from 'react-firebase-hooks/database'
 
+if (!firebase.apps.length) {
+  firebase.initializeApp(config)
+}else {
+  firebase.app()
+}
 type Props = {
 }
 
 const Watch: FC<Props> = () => {
   const { query } = useRouter()
```

Comment:
We may have to account for SSR/G once this is deployed but I'm not sure. If we have issues, we can try some things to disable SSR/G